### PR TITLE
Added links to resource specs in readme

### PR DESCRIPTION
**Pull request by a-thaler**
**Description**

Changes proposed in this pull request:

- added links to the resource specs to readme
- simplified usage in readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,66 +30,20 @@ Perform the following steps to build the providers:
 
 ## Usage
 
-Perform the following steps to use the provider:
+At the moment the resource type [`gardener_shoot`](shoot/schema_shoot.go) is supported corresponding to the [`shoot`](https://github.com/gardener/gardener/blob/master/pkg/apis/core/v1alpha1/types_shoot.go) resource of gardener. The cloud provider specific configuration and a related example is listed in the table below:
```
Comment (bszwarc) [inline README.md]:
```suggestion
At the moment, the Terraform Provider for Gardener defines a resource type [`gardener_shoot`](shoot/schema_shoot.go) which is linked to the corresponding [`shoot`](https://github.com/gardener/gardener/blob/master/pkg/apis/core/v1alpha1/types_shoot.go) Gardener resource. The table shows the cloud provider-specific configuration and related examples:
```

File: README.md [hunk 2]
```diff
@@ -30,66 +30,20 @@ Perform the following steps to build the providers:
...
+|---|---|---|
+|Amazon Web Services|[schema.go](shoot/schema_aws.go)|[main.tf](examples/aws/main.tf)|
+|Goocle Cloud Platform|[schema.go](shoot/schema_gcp.go)|[main.tf](examples/gcp/main.tf)|
+|Azure|[schema.go](shoot/schema_azure.go)|[main.tf](examples/azure/main.tf)|
 
-    ```bash
-    cd examples/<provider>
-    ```
-2. Edit the `main.tf` file to provide the following parameters:
+-Perform the following steps to use the provider:
```
Comment (bszwarc) [inline README.md]:
```suggestion
Perform the following steps to use the provider:
```

File: README.md [hunk 3]
```diff
@@ -30,66 +30,20 @@ Perform the following steps to build the providers:
...
+ 
+1. Navigate to one of the `main.tf` files as listed above
```
Comment (bszwarc) [inline README.md]:
```suggestion
1. Go to one of the `main.tf` files listed above.
```

File: README.md [hunk 4]
```diff
@@ -30,66 +30,20 @@ Perform the following steps to build the providers:
...
 
-    - Path to the Gardener kubeconfig
-    - Shoot specification
+2. Edit the `main.tf` file to provide the missing parameters
```
Comment (bszwarc) [inline README.md]:
```suggestion
2. Edit the `main.tf` file to provide the missing parameters.
```

File: examples/README.md [hunk 5]
```diff
@@ -37,6 +37,8 @@ kube_file          =<<-EOT
 This section includes the following parameters:
 * **kube_file** - the raw string of the kube config file.
 
+> **NOTE:** To obtain the gardener secret and kubeconfig go to the [Gardener dashboard](https://dashboard.garden.canary.k8s.ondemand.com/login).
```
Comment (bszwarc) [inline examples/README.md]:
```suggestion
>**NOTE:** To obtain the Gardener secret and the `kubeconfig` file go to the [Gardener dashboard](https://dashboard.garden.canary.k8s.ondemand.com/login).
```

File: examples/gcp/README.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 # Deploy a cluster on GCP
 
 ## Overview
-Use this configuration example to deploy a GCP cluster using Gardener.
+Use the configuration example located in [main.tf](main.tf) to deploy a GCP cluster using Gardener.
```
Comment (bszwarc) [inline examples/gcp/README.md:4]: Should it be a GCP or GKE cluster?

**Review by bszwarc — CHANGES_REQUESTED** [review]
Added a few comments and a question.

File: README.md (hunk 1 above)
Comment (a-thaler) [reply README.md]: That's not what I wanted to say. This terraform provider defines a resource type "shoot" which will then be transformed and applied to Gardener. On gardener side there is a corresponding resource which we are mapping to. I wanted to link the mapped gardener type. So it is not a "supported by" relation..

File: examples/gcp/README.md (hunk 6 above)
Comment (a-thaler) [reply examples/gcp/README.md:4]: It is a kubernetes cluster on GCP not using GKE, so it is fine

File: go.mod [hunk 7]
```diff
@@ -14,14 +14,12 @@ require (
 	github.com/mattn/go-isatty v0.0.9 // indirect
```
Comment (clebs) [inline go.mod:14]: Maybe we can use this PR to update terraform to `0.12.13` :)

**Review by clebs — APPROVED** [review]

File: go.mod (hunk 7 above)
Comment (a-thaler) [reply go.mod:14]: done

File: README.md (hunk 1 above)
Comment (bszwarc) [reply README.md]: Can we maybe use the word "linked"?

**Review by bszwarc — APPROVED** [review]

**Pull request merged**